Project.toml

```diff
@@ -5,6 +5,7 @@ version = "0.1.0"
 
 [deps]
 CTBase = "54762871-cc72-4466-b8e8-f6c8b58076cd"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
I would prefer we remove DataFrames.jl from the deps

Reply:
But he needs it for the metadata. What do you suggest?

Reply:
Actually, he did the same as here:

https://github.com/JuliaSmoothOptimizers/OptimizationProblems.jl/blob/2c14b0296eebae606a86edcbcfebb2b7faa0bb21/src/OptimizationProblems.jl#L84

Reply:
I would just use dictionary to store the metadata, I don't think we need to use DataFrames for that